saleor/product/migrations/0187_category_updated_at.py

```diff
@@ -0,0 +1,17 @@
+# Generated by Django 3.2.20 on 2023-08-30 06:01
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("product", "0186_remove_product_charge_taxes"),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name="category",
+            name="updated_at",
+            field=models.DateTimeField(auto_now=True, db_index=True),
```

Comment:
I think we need to make a db index concurrently here. 
Something like here: https://github.com/saleor/saleor/pull/13707/files
@kadewu  Can you help?

Reply:
Also, the `updated_at` would need to be nullable to provide zero-downtime migration: https://docs.saleor.io/docs/next/developer/community/zero-downtime-migrations

Reply:
As soon as we get an additional feedback concerning a db index, I will recreate a migrations so that it contains a nullable in accordance with zero-downtime migration.

Reply:
Hi @rafiwts, in case of indexes, you need to use [`Meta.indexes`](https://docs.djangoproject.com/en/3.2/ref/models/options/#django.db.models.Options.indexes) over `db_index` and in migration switch to `concurrently` option: https://docs.saleor.io/docs/next/developer/community/zero-downtime-migrations#adding-index-to-the-database

Reply:
The new migrations has been implemented. See a comment below.